tests/test_inclined_exponential.py

```diff
@@ -126,211 +281,223 @@ def test_edge_on():
+                get_prof(mode,0.1*galsim.radians, 3.1, 0.3),
+                get_prof(mode,0.1*galsim.radians, 3.1),
+                get_prof(mode,0.1*galsim.radians, 3.0, 0.3, flux=0.5),
+                get_prof(mode,0.1*galsim.radians, 3.0, 0.3, gsparams=gsp)]
+        all_obj_diff(diff_gals)
 
 @timer
 def test_pickle():
     """ Check that we can pickle it. """
-
-    exp = galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                     scale_height=0.3)
-    do_pickle(exp)
-    do_pickle(exp.SBProfile)
-    do_pickle(galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0))
-    do_pickle(galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                         scale_h_over_r=0.2))
-    do_pickle(galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                         scale_height=0.3, flux=10.0))
-    do_pickle(galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                         scale_height=0.3,
-                                         gsparams=galsim.GSParams(folding_threshold=1.1e-3)))
-    do_pickle(galsim.InclinedExponential(inclination=0.1*galsim.radians, scale_radius=3.0,
-                                         scale_height=0.3, flux=10.0,
-                                         gsparams=galsim.GSParams(folding_threshold=1.1e-3)))
+    
+    for mode in ("InclinedExponential","InclinedSersic"):
+
+        prof = get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0,
+                                         scale_height=0.3)
+        do_pickle(prof)
+        do_pickle(prof.SBProfile)
+        do_pickle(get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0))
+        do_pickle(get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0,
+                                             scale_h_over_r=0.2))
+        do_pickle(get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0,
+                                             scale_height=0.3, flux=10.0))
+        do_pickle(get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0,
+                                             scale_height=0.3,
+                                             gsparams=galsim.GSParams(folding_threshold=1.1e-3)))
+        do_pickle(get_prof(mode,trunc=4.5,inclination=0.1*galsim.radians, scale_radius=3.0,
+                                             scale_height=0.3, flux=10.0,
+                                             gsparams=galsim.GSParams(folding_threshold=1.1e-3)))
 
 if __name__ == "__main__":
-    test_regression()
-    test_exponential()
-    test_edge_on()
     test_sanity()
     test_k_limits()
     test_eq_ne()
     test_pickle()
+    test_exponential()
+    test_sersic()
+    test_edge_on()
+    test_regression()
```

Comment:
Extremely minor point, but we usually keep these in the same order that they appear in the file.